usage-rules.md

```diff
@@ -1131,3 +1131,62 @@ When testing resources:
 - Use `authorize?: false` in tests where authorization is not the focus
 - Write generators using `Ash.Generator`
 - Prefer to use raising versions of functions whenever possible, as opposed to pattern matching
+
+### Preventing Deadlocks in Concurrent Tests with Ash.Generator
+
+When running tests concurrently, using fixed values for identity attributes can cause postgres deadlock errors. Multiple tests attempting to create records with the same unique values will conflict. Ash provides `Ash.Generator` with built-in utilities for generating unique test data.
+
+#### Using Ash.Generator.sequence/3
```

Comment:
Since this is unique only on a per-process basis, I'm not entirely sure we should include it here in this form. I mean, all the info is here, but I'm not sure if only having the System.unique_integer might be better. WDYT? :thinking:

Reply:
I agree, this will probably make it wrong more often.